default/scripting/buildings/INTERSPECIES_ACADEMY.focs.txt

```diff
@@ -76,32 +77,8 @@ BuildingType
             activation = Source
             accountinglabel = "INTERSPECIES_ACADEMY_LABEL"
             priority = [[LATE_PRIORITY]]
-            effects = [
```

Comment:
@agrrr3 not really sure what this building is supposed to do or should do with this setup... alternatives requested.

Reply:
In this branch each academy boosts the local max contribution to the global limit like the other tech but without needing the focus set to stockpile. I think this as okayish.

Compared to tech the idea of the building is that is a more natural fit for distributed empires (or large empires) because you need space between the buildings. I would like to have such a tech in the game.

One could gate the effect also by focus, but this would make it less distinguished from the other techs. With the other techs you can make any number of planets contribute, that is not possible with the academy. Not sure on this one.

Another use would be to make the species on the planet a better stockpiling species (so that the whole population of that in the species would contribute more), then we would need to make the building a lot more expensive probably. And to make it more strategic, we could make the effect interruptible. Only boost the the stockpiling if there were no enemy ships in orbit the last turn.

Or we could make it a research bonus instead of stockpiling bonus, because exchange between people and ideas is how research flourishes. Then we need to remove the "design" part from the fluff, because design is not about research but about application.

Reply:
> Or we could make it a research bonus instead of stockpiling bonus, because exchange between people and ideas is how research flourishes. Then we need to remove the "design" part from the fluff, because design is not about research but about application.

Sorry for chiming in only now, but I really want to voice my support for this idea. It just makes so much more sense for a building like this...